server/src/main/java/org/opensearch/gateway/BaseGatewayShardAllocator.java

```diff
@@ -120,6 +158,18 @@ public abstract AllocateUnassignedDecision makeAllocationDecision(
         Logger logger
     );
 
+    public HashMap<ShardRouting, AllocateUnassignedDecision> makeAllocationDecision(
+        Set<ShardRouting> shards,
```

Comment:
Rename the `shards` param here ? One recommendation is `unassignedShardBatch`.

Reply:
ack